iati/core/constants.py

```diff
@@ -9,7 +9,7 @@
     Allow logging constants to be user-definable.
 """
 
-STANDARD_VERSIONS = ['2.02']
+STANDARD_VERSIONS = ['1.04', '1.05', '2.01', '2.02']
```

Comment:
There should probably be a test checking the length of this against a constant (since its length is used in a number of places in a dynamic manner).

Reply:
Added in a98f93e